backend/attendance-service/src/main/java/org/egov/repository/AttendeeRepository.java

```diff
@@ -0,0 +1,40 @@
+package org.egov.repository;
+
+import org.apache.commons.lang3.StringUtils;
+import org.egov.repository.rowmapper.AttendeeQueryBuilder;
+import org.egov.repository.rowmapper.AttendeeRowMapper;
+import org.egov.repository.rowmapper.StaffQueryBuilder;
+import org.egov.repository.rowmapper.StaffRowMapper;
+import org.egov.web.models.AttendanceRegister;
+import org.egov.web.models.AttendanceRegisterSearchCriteria;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Repository
+public class AttendeeRepository {
+
+    @Autowired
+    private AttendeeQueryBuilder attendeeQueryBuilder;
+
+    @Autowired
+    private AttendeeRowMapper attendeeRowMapper;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public List<AttendanceRegister> getRegisteredAttendees(AttendanceRegisterSearchCriteria searchCriteria) {
```

Comment:
Method name is not reflecting the purpose of the method. Applies to other methods also.

Reply:
The method returns the attendees registered to the attendance register. So it is getRegisteredAttendees. The List<AttendanceRegister> object will have the register and the corresponding attendees details. Same goes for the getRegisteredStaffs which have the corresponding staff details.